firstvoices/firstvoices/settings.py

```diff
@@ -83,6 +83,9 @@
     "UNAUTHENTICATED_USER": None,
     "DEFAULT_PAGINATION_CLASS": "backend.pagination.PageNumberPagination",
     "PAGE_SIZE": 100,
+    "JSON_UNDERSCOREIZE": {
+        "ignore_fields": ("site_data_export",),
+    },
```

Comment:
The purpose of this is to ignore the automatic camel casing in the API and keep the underscore format needed for Mother Tongues. See https://github.com/vbabiy/djangorestframework-camel-case#ignore-fields for more info.